nest-backend/src/decorators/host.decorator.ts

```diff
@@ -0,0 +1,8 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common'
+
+export const ServerHost = createParamDecorator((data: unknown, ctx: ExecutionContext) => {
+  const req = ctx.switchToHttp().getRequest()
+  const url = req.protocol + '://' + req.get('host')
```

Comment:
As discussed previously we use this format of string concats

```suggestion
  const url = `${req.protocol}://${req.get('host')}`
```